packages/twenty-front/src/modules/object-record/record-table/record-table-cell/components/RecordTableCellCheckbox.tsx

```diff
@@ -33,6 +33,7 @@ export const RecordTableCellCheckbox = () => {
 
   const handleClick = useCallback(
     (event?: React.MouseEvent<HTMLDivElement>) => {
+      event?.stopPropagation();
```

Comment:
using stopPropagation is usually not a good idea as this is very sensible to dom changes. We prefer using the notion of focusStack to understand what element should have the focus